# goal definition
assignment_1/target_ids target_ids  # List of IDs to find
---
# result definition
bool success  # True if all target_ids were found
int32[] found_ids  # IDs found at the end
geometry_msgs/PoseStamped[] found_poses  # Position of found IDs
---
# feedback
int32 detected_id  # IDs found while moving
geometry_msgs/PoseStamped detected_pose  # Relative position